charts/oidc-webhook-authenticator/values.yaml

```diff
@@ -20,7 +20,11 @@ autoscaling:
     cpuTargetAverageUtilization: 80
 
 webhookConfig:
-  tls:
+  certificates:
+    # If certManager.enabled is set to `true` then other configurations in the `certificates` section will be ignored.
+    certManager:
+      enabled: false
```

Comment:
The subcharts `application` and `runtime` can now be properly configured via their respective sections in the `values.yaml` file. I also added support for conditional installation of the subcharts. For example if the installation of the `application` subchart is not required this can be achieved by passing `--set application.enabled=false` flag during installation.